src/Microsoft.VisualStudio.ProjectSystem.Managed.VS.UnitTests/Mocks/IUnconfiguredProjectVsServicesFactory.cs

```diff
@@ -35,6 +35,12 @@ public static IUnconfiguredProjectVsServices Implement(Func<IVsHierarchy> hierar
                     .Returns(projectCreator());
             }
 
+            if (configuredProjectCreator != null)
```

Comment:
Didn't end up needing this in the end, but I don't think it causes any harm being here?